src/gcp_scanner/crawl.py

```diff
@@ -703,36 +703,37 @@ def list_services(project_id: str, service: discovery.Resource) -> List[Any]:
   return list_of_services
 
 
-def list_sourcerepo(project_id: str,
-                    service: discovery.Resource) -> List[Any]:
-  """Retrieve a list of cloud source repositories enabled in the project.
 
+def list_dns_policies(project_id: str,
```

Comment:
The rest of the change is about source repos but this lists dns policies? Did you mix up the refactorings?

Reply:
Added the dns_policies to remove the merge conflict caused by #214

Reply:
Apologies, I didn't realize the merge conflict was caused by #214 - updated crawl.py to match main